ldk/javascript/examples/self-test-loop/src/tests/whisper/index.ts

```diff
@@ -2054,7 +2054,7 @@ export const testRichTextEditor = (): Promise<boolean> =>
               onClick: (_error: Error, onClickWhisper: Whisper) => {
                 if (!editedText || editedText.length === 0 || editedText.length > 50) {
                   (components[0] as RichTextEditor).validationError =
-                    'Inputed text is required and should be less than 200 chars.';
+                    'Inputed text is required and should be less than 50 chars.';
```

Comment:
I am out of the loop on this change - Is the character limit already limited to 50 characters in the sidekick app?

Reply:
@cshields-oliveai  We created a quick sample to show validation error, the logic is on row 2056 in this file. `if (!editedText || editedText.length === 0 || editedText.length > 50) {
                  (components[0] as RichTextEditor).validationError =
                    'Inputed text is required and should be less than 200 chars.';`

I just need to update the error message content.